src/shaders/colorimetry.h

```diff
@@ -96,36 +96,36 @@ struct PrimaryInfo {
 };
 
 vec3 convert_primary(vec2 xy) {
-	float X = xy.x / xy.y;
-	float Y = 1.0f;
-	float Z = (1.0f - xy.x - xy.y) / xy.y;
-	return vec3(X, Y, Z);
+    float X = xy.x / xy.y;
+    float Y = 1.0f;
+    float Z = (1.0f - xy.x - xy.y) / xy.y;
+    return vec3(X, Y, Z);
 }
 
 mat3 compute_xyz_matrix(PrimaryInfo metadata) {
-	vec3 red = convert_primary(metadata.displayPrimaryRed);
```

Comment:
lots of random whitespace changes in this file, pls fix

Reply:
the file uses 4 spaces as indentation and these were using tab as indentation. the VScode indentation conversion from tabs to 4 spaces got them when I converted my code which used tabs as indentation

Reply:
Yes, please fix the random whitespace changes, it makes viewing diffs really annoying.